test/pkcs12_api_test.c

```diff
@@ -157,7 +292,9 @@ int setup_tests(void)
 
     ADD_TEST(test_null_args);
     ADD_TEST(pkcs12_parse_test);
-
+    ADD_TEST(pkcs12_create_ex2_success_test);
+    ADD_TEST(pkcs12_create_ex2_failure_test);
+    ADD_TEST(pkcs12_create_ex2_error_test);
```

Comment:
Since all of these 3 tests have more-or-less the same implementations with minor differences I would instead combine this into one function and use `ADD_ALL_TESTS`